README.md

```diff
@@ -114,10 +114,22 @@ $ ./m8c
 INFO: Looking for USB serial devices.
 INFO: Found M8 in /dev/ttyACM1.
 INFO: Opening port.
-Aborted (core dumped)
+ERROR: Error: Failed: Permission denied
```

Comment:
Now that the error messages are displayed, this is the error that came out on my system.